server/app/controllers/FileController.java

```diff
@@ -117,7 +121,9 @@ private Result adminShowInternal(Request request, String fileKey) {
     AccountModel adminAccount =
         profileUtils.currentUserProfile(request).orElseThrow().getAccount().join();
 
-    return maybeFile.get().getAcls().hasProgramReadPermission(adminAccount)
+    return ((adminAccount.getGlobalAdmin()
```

Comment:
(non-blocking) Maybe add a comment on the business rules explaining what this is checking.

Reply:
Good call - added!